style.css

```diff
@@ -1,818 +1,682 @@
-  font-size: 16px;
-  color: #fff;
-  text-align: center;
+.header-container {
+  width: 90vw;
+  display: flex;
+  flex-direction: row-reverse;
+  gap: 12px;
+  padding-top: 4px;
+  padding-bottom: 4px;
 }
 
-.blackBoton {
-  font-family: 'Poppins', sans-serif;
-  margin: 16px 16px;
-  width: 90%;
-  height: 50px;
-  background: #1a2236;
+.list-container {
+  display: flex;
+  list-style: none;
   color: #fff;
-  margin-bottom: 32px;
+  justify-content: space-evenly;  /* ----------- */
+  min-width: 200px;
+  margin-top: 0;
+  margin-bottom: 0;
+  margin-right: 10px;
+  padding-top: 5px;
 }
 
-.imagebox {
-  display: block;
-  height: 160px;
-  background-image: url(images/image_geometry_2.png);
+.header-tw {
+  max-height: 30px;
 }
 
-/* projects section */
+.header-fb {
+  max-height: 28px;
+}
 
-.gallery {
-  display: grid;
-  flex-wrap: wrap;
-  width: 100vw;
+#logo {
+  max-height: 50px;
 }
 
-h2 {
-  font-family: 'Playfair Display', serif;
-  font-size: 3.5em;
-  font-weight: 400;
-  line-height: 100%;
-  padding: 16px;
-  padding-bottom: 0;
-  padding-top: 0;
-  margin: 40px 0 2rem;
-  background: #fff;
-  background-color: transparent;
+.nav-row {
+  padding-top: 5px;
+  padding-bottom: 5px;
+  background-color: #fff;
+  margin-top: 0;
+  margin-left: 0;
 }
 
-.card {
+.nav {
   display: flex;
-  flex-wrap: wrap;
-  width: 100vw;
+  flex-direction: row;
+  max-width: 100vw; /* ------- */
+  justify-content: space-around;
 }
 
-.pData {
-  width: 100%;
-  height: 156px;
-  background: #1a2236;
-  padding-left: 16px;
-  bottom: 0;
+.nav-menu-desktop {
+  display: flex;
+  justify-content: space-between;   /* ---- */
+  min-width: 470px;
+  margin-left: 10%;
 }
 
-h3 {
-  font-family: 'Poppins', sans-serif;
-  font-weight: 100;
-  font-size: 20px;
-  line-height: 300%;
-  color: #fff;
+.nav-menu-mobile {
+  display: none;
 }
 
-.projectUl {
+.menu-items-desktop {
   display: flex;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 300;
-  list-style-type: none;
-  padding: 0;
-  padding-left: 0;
-  margin-bottom: 0;
-  width: 314px;
-}
-
-.projectLi {
-  font-size: 10.33px;
-  background-color: #98dede;
-  float: left;
-  text-align: center;
-  margin-right: 10px;
-  padding: 6px;
-  padding-top: 1.5px;
-  padding-bottom: 1.5px;
+  list-style: none;
+  justify-content: space-between;   /* ---- */
+  min-width: 300px;
 }
 
-.see {
-  background: none;
-  border: #fff solid 0.5px;
-  margin-top: 20px;
-  margin-bottom: 20px;
-  padding: 7px 10px;
-  color: #dddfe6;
+.header-button {
+  background-color: transparent;
+  border-style: solid;
+  border-color: #ec5242;
+  border-width: 4px;
+  color: #ec5242;
+  margin-left: 8%;
 }
 
-/* about section */
+/* ---------------iNTRO-SECTION------------ */
 
-.about {
+.intro-container {
   display: flex;
-  flex-direction: column;
+  flex-direction: row;
+  justify-content: center;
+  background-color: #eeecec;
+  opacity: 0.96;
+  margin: 0;
 }
```

Comment:
- [ ] As mentioned previously please make sure you use a background that's similar to the given design in overall appearance because as you can see from the original design the background color is white with some semi-transparent properties and a couple of icons. so please make sure you implement this instead of using a solid color like `#eeecec`